pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -142,6 +143,15 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: option %q specified for HairpinMode (--hairpin-mode). Valid options are %q, %q or %q",
 			kc.HairpinMode, kubeletconfig.HairpinNone, kubeletconfig.HairpinVeth, kubeletconfig.PromiscuousBridge))
 	}
+	if kc.ReservedSystemCPUs != "" {
+		// --reserved-cpus does not support --system-reserved-cgroup or --kube-reserved-cgroup
+		if kc.SystemReservedCgroup != "" || kc.KubeReservedCgroup != "" {
+			allErrors = append(allErrors, fmt.Errorf("can't use --reserved-cpus with --system-reserved-cgroup or --kube-reserved-cgroup"))
+		}
+		if _, err := cpuset.Parse(kc.ReservedSystemCPUs); err != nil {
+			allErrors = append(allErrors, fmt.Errorf("invalid configuration: --reserved-cpus must be in a string of digits separated by comma or -, not %q", kc.ReservedSystemCPUs))
```

Comment:
shouldn't we include the parse error in the validation error message?

Reply:
updated

Reply:
@liggitt ok to proceed?